docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncodingMultiColumn.cs

```diff
@@ -2,44 +2,53 @@
-            var samples = new List<DataPoint>()
+            var samples = new List<DataPoint>
             {
-                new DataPoint(){ Education = "0-5yrs", ZipCode = "98005" },
-                new DataPoint(){ Education = "0-5yrs", ZipCode = "98052" },
-                new DataPoint(){ Education = "6-11yrs", ZipCode = "98005" },
-                new DataPoint(){ Education = "6-11yrs", ZipCode = "98052" },
-                new DataPoint(){ Education = "11-15yrs", ZipCode = "98005" },
+                new DataPoint {Education = "0-5yrs", ZipCode = "98005"},
+                new DataPoint {Education = "0-5yrs", ZipCode = "98052"},
+                new DataPoint {Education = "6-11yrs", ZipCode = "98005"},
+                new DataPoint {Education = "6-11yrs", ZipCode = "98052"},
+                new DataPoint {Education = "11-15yrs", ZipCode = "98005"}
             };
 
             // Convert training data to IDataView.
-            var data = mlContext.Data.LoadFromEnumerable(samples);
+            IDataView data = mlContext.Data.LoadFromEnumerable(samples);
 
-            // Multi column example : A pipeline for one hot encoding two columns 'Education' and 'ZipCode' 
-            var multiColumnKeyPipeline = mlContext.Transforms.Categorical.OneHotEncoding(
-                new InputOutputColumnPair[] {
-                    new InputOutputColumnPair("Education"),
-                    new InputOutputColumnPair("ZipCode"),
-                });
+            // Multi column example : A pipeline for one hot encoding two columns
+            // 'Education' and 'ZipCode' 
+            var multiColumnKeyPipeline =
+                mlContext.Transforms.Categorical.OneHotEncoding(
+                    new[]
+                    {
+                        new InputOutputColumnPair("Education"),
+                        new InputOutputColumnPair("ZipCode")
+                    });
 
             // Fit and Transform data.
-            var transformedData = multiColumnKeyPipeline.Fit(data).Transform(data);
+            IDataView transformedData =
+                multiColumnKeyPipeline.Fit(data).Transform(data);
 
-            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, true);
+            var convertedData =
+                mlContext.Data.CreateEnumerable<TransformedData>(transformedData,
+                    true);
 
-            Console.WriteLine("One Hot Encoding of two columns 'Education' and 'ZipCode'.");
-            foreach (var item in convertedData)
-                Console.WriteLine("{0}\t\t\t{1}", string.Join(" ", item.Education), string.Join(" ", item.ZipCode));
+            Console.WriteLine(
+                "One Hot Encoding of two columns 'Education' and 'ZipCode'.");
+
+            foreach (TransformedData item in convertedData)
+                Console.WriteLine("{0}\t\t\t{1}", string.Join(" ", item.Education),
+                    string.Join(" ", item.ZipCode));
```

Comment:
Maybe use interpolation? Easier to read. Unless we decided to not do that at some point.

Reply:
I'm not sure what you mean but this is the general pattern we use when we have to print a list.

---
In reply to: [278758746](https://github.com/dotnet/machinelearning/pull/3588#discussion_r278758746) [](ancestors = 278758746)